VerificationWeb/Views/Shared/_Layout.cshtml

```diff
@@ -12,12 +12,13 @@
     </environment>
     <environment exclude="Development">
         <link rel="stylesheet" href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.5.0/fedora-bootstrap.min.css"
-              crossorigin="anonymous"
-              integrity="sha384-sP7GRmaXVgvziloAQfY0eqn1lWHf55f7EWQwxEYh/iAIUzcUZr9olNPo1d/J9vm6"/>
+              @* Fedora Apps doesn't currently have cors enabled *@
+              @* crossorigin="anonymous" *@
+              @* integrity="sha384-sP7GRmaXVgvziloAQfY0eqn1lWHf55f7EWQwxEYh/iAIUzcUZr9olNPo1d/J9vm6" *@/>
```

Comment:
It's the same issue, I don't think that you can use anything directly from fedoraproject.org because of cors


https://stage-verify.valkyrja.app/

Reply:
May need to copy it locally or I don't know (probably also better - won't break our thing if they change something?)

Reply:
The releases should be stable, so nothing there should change without a version change